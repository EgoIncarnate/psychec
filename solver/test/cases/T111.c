
void f() {
    const char* s;
    char const* s1;
    const char* const s2;
    g(s);
    h(s1);
    j(s2);

    const T1 x;
    T1 const y;
    T1 z;
    k(x);
    l(y);
    m(z);
    m(x);
}

