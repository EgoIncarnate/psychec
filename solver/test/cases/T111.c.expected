typedef uintptr_t const T1 ;
uintptr_t g (char const*) ;
uintptr_t h (char const*) ;
uintptr_t j (char const* const) ;
uintptr_t k (uintptr_t const) ;
uintptr_t l (uintptr_t const) ;
uintptr_t m (uintptr_t const) ;
